src/components/common/SendCode/layouts/RequestCode.vue

```diff
@@ -27,6 +27,7 @@ export default {
 
   methods: {
     onClick() {
+      this.$emit('update:is-locked', true);
       this.$emit('click');
```

Comment:
no need to emit `update:is-locked`
and maybe change `click` to `send-code` to see/read the `intention`